src/cover-traffic.md

```diff
@@ -13,3 +13,5 @@ Nodes should dispatch drop and loop cover packets in each active session accordi
 processes. The average drop/loop rates should be the same for all mixnodes in a session, and for
 all non-mixnodes in a session; see the [parameters chapter](./parameters.md). Note that the rates
 should be halved in some phases; see the [sessions chapter](./sessions.md#phases).
+
+TODO maybe be explicit if `Node` is `mixnet-node` or if it includes `non-mixnet-nodes`.
```

Comment:
I can try rewording this. I thought it was clear enough that it meant both given that it mentions both mixnodes and non-mixnodes?

Reply:
I am not sure what to do if I am no t a mixnet node, problably I should generate cover traffic, but if all non mixnet node generate cover traffic, then I think we got a problem (I am thinking of non mixnet node as an unbound resource but if we consider them as bounded by the number of accepted node by validator it can be ok, but then we got a problem of node blocking access to others).
So I was more thinking that the sentence did apply only for the mixnet node.
My understanding was that non mixnet node are allowed to do cover and should probably do if they are going to send a real message, but no rules could really apply for them.

Reply:
All nodes that are connected to the mixnet are expected to generate cover traffic. Ideally we want some nodes that aren't intending to send messages to connect to the mixnet anyway, but we almost certainly don't want _all_ nodes to connect to the mixnet as this would overload it. I don't really have a solution to this at the moment. Perhaps nodes not intending to send messages could connect with some probability determined by the runtime. This would probably be quite slow to react to changes in the total number of nodes, but if that number is fairly stable it could work.

Reply:
For the initial version, simply enabling the mixnet by default for nodes run with `--validator` could work.

Reply:
FWIW the cover traffic rate for non-mixnodes is currently 10x lower than for mixnodes. N mixnodes should comfortably support 10*N non-mixnodes.

Reply:
Thanks, makes sense, seems important to have those numbers on doc (even if nobody is forced to emit cover messages it can give an idea of the possible behavior).